.github/workflows/stress-test.yml

```
@@ -18,7 +18,7 @@ jobs:
         run: rustup update stable
 
       - name: Install Valgrind
-        run: apt install -y valgrind
+        run: sudo apt install -y valgrind
```

```suggestion
        run: sudo apt-get install -y valgrind
```

https://github.com/tokio-rs/tokio/runs/1515308234?check_suite_focus=true

> WARNING: apt does not have a stable CLI interface. Use with caution in scripts.